scripts/cesm3_LTso_workflow_TEMPLATE.sh

```diff
@@ -0,0 +1,338 @@
+'TTGWSPEC', 'U', 'U10', 'UTGWORO', 'UTGWSPEC', 'UVzm', 'UWzm', 'Uzm', 'V', 'VD01', 'VTHzm', 
+'Vzm', 'WD_NH3', 'WD_NH4', 'WD_NOY', 'WD_SO2', 'Wzm', 'Z3', 'ZMDQ', 'ZMDT', 'ZMMTT', 'ZMMU'
+
+EOF
+
+
+if [[ $do_dy == true ]]; then
+cat <<EOF>> user_nl_cam
+
+! daily
+fincl2 = 'ACTNL', 'ACTREL', 'AODVISdn', 'bc_a1', 'bc_a4', 'BURDENBCdn', 'BURDENDUSTdn', 'BURDENPOMdn', 
+'BURDENSEASALTdn', 'BURDENSO4dn', 'BURDENSOAdn', 'BUTGWSPEC', 'CDNUMC', 'CLD_CAL', 'CLDHGH_CAL', 'CLDICE', 
+'CLDLIQ', 'CLDLOW_CAL', 'CLDMED_CAL', 'CLDTOT', 'CLDTOT_CAL', 'CLDTOT_ISCCP', 'CLOUD', 'CO', 'DF_ALKNIT', 
+'DF_HNO3', 'DF_HO2NO2', 'DF_HONITR', 'DF_ISOPNITA', 'DF_ISOPNITB', 'DF_NH3', 'DF_NH4', 'DF_NO', 'DF_NO2', 
+'DF_NOA', 'DF_ONITR', 'DF_PAN', 'DF_TERPNIT', 'dst_a1', 'dst_a2', 'dst_a3', 'EFLX_LH_TOT', 'EFLX_LH_TOT_ICE', 
+'FCTL', 'FISCCP1_COSP', 'FLASHFRQ', 'FLDS', 'FLDSC', 'FLNS', 'FLUC', 'FLUT', 'FLUTC', 'FSA', 'FSCSC', 'FSDS', 
+'FSDSC', 'FSH_ICE', 'FSNS', 'FSNSC', 'FSNTOA', 'FSNTOAC', 'FSR_ICE', 'FSUTOA', 'LHFLX', 'MEANCLDALB_ISCCP', 
+'MEANPTOP_ISCCP', 'ncl_a1', 'ncl_a2', 'ncl_a3', 'ncl_c1', 'ncl_c2', 'ncl_c3', 'NH4', 'O3_SRF', 'OMEGA', 'OMEGA500', 
+'PM25_SRF', 'pom_a1', 'pom_a4', 'PR', 'PRECC', 'PRECT', 'PS', 'PSIRS', 'PSL', 'PTTEND', 'Q', 'QFLX', 'QFLX_EVAP_TOT', 
+'QFLX_SNOW_DRAIN', 'QREFHT', 'QRL', 'QRS', 'QSNOFRZ', 'QSNOW', 'RAIN', 'RAIN_ICE', 'RELHUM', 'RHREFHT', 'SHFLX', 
+'SNOCAN', 'SNOW', 'SO2', 'so4_a1', 'so4_a2', 'so4_a3', 'soa1_a1', 'soa1_a2', 'soa2_a1', 'soa2_a2', 'soa3_a1', 
+'soa3_a2', 'soa4_a1', 'soa4_a2', 'soa5_a1', 'soa5_a2', 'SOLIN', 'SOLLD', 'SOLSD', 'T', 'T700', 'T850', 'TAUBLJX', 
+'TAUBLJY', 'TAUGWX', 'TAUGWY', 'TAUX', 'TAUY', 'TGCLDIWP', 'TGCLDLWP', 'THzm', 'TMO3', 'TMQ', 'TOT_CLD_VISTAU', 
+'TREFHT', 'TREFHTMN', 'TREFHTMX', 'TREFMNAV', 'TREFMXAV', 'TS', 'U', 'U10', 'UTGWORO', 'UTGWSPEC', 'UVzm', 'UWzm', 
+'Uzm', 'V', 'VTHzm', 'Vzm', 'WD_NH3', 'WD_NH4', 'WD_NOY', 'Wzm', 'Z1000', 'Z3', 'Z500', 'ZMFLXSNW'
+
+EOF
+
+fi
+
+if [[ $do_hr6 == true ]]; then
+cat <<EOF>> user_nl_cam
+
+! 6 hourly
+fincl3 = 'FLDS', 'FSDS', 'PRECT', 'PSL', 'PS', 'QREFHT', 'RAIN', 'SNOW', 'RHREFHT', 
+'TMQ', 'TREFHT', 'TS', 'U', 'U10', 'V', 'Z1000', 'Z3', 'Z500', 'Z700', 'Z925', 
+'PRECTMX', 'RELHUM100', 'RELHUM500', 'RELHUM850', 'Q', 'UREFHT', 'VREFHT'
+
+EOF
+
+fi
+
+if [[ $do_hr3 == true ]]; then
+cat <<EOF>> user_nl_cam
+
+! 3 hourly
+fincl4 = 'QREFHT', 'PRECT', 'SNOW', 'PS', 'FLDS', 'FSDS', 'TREFHT', 'RHREFHT', 
+'num_a1', 'num_a2', 'num_a3', 'num_a4', 'bc_a1', 'bc_a4', 'dst_a1', 'dst_a2', 'dst_a3', 
+'NH4', 'pom_a1', 'pom_a4', 'soa1_a1', 'soa1_a2', 'soa2_a1', 'soa2_a2', 'soa3_a1', 
+'soa3_a2', 'soa4_a1', 'soa4_a2', 'soa5_a1', 'soa5_a2', 'so4_a1', 'so4_a2', 'so4_a3', 
+'ncl_a1', 'ncl_a2', 'ncl_a3', 'ncl_c1', 'ncl_c2', 'ncl_c3', 'SO2', 'U10'
+
+EOF
+
+fi
+
+########################
+## user_nl_clm
+########################
+
+cat <<EOF> user_nl_clm
+reseed_dead_plants = .true.
```

Comment:
I'm not sure that you always want to have the reseed_dead_plants option on for all experiments. This might be something that we should check on with the CTSM peeps about.